_posts/2020-07-21-cube-functionality-to-kartothek.markdown

```diff
@@ -0,0 +1,414 @@
+0         8                   6.0   Dresden      DE 2018-01-01  51.050407  13.737262
+1         4                   4.0   Dresden      DE 2018-01-02  51.050407  13.737262
+2         6                   6.0   Hamburg      DE 2018-01-01  53.551086   9.993682
+3         5                   4.0   Hamburg      DE 2018-01-02  53.551086   9.993682
+4         6                   6.0    London      UK 2018-01-01  51.509865  -0.118092
+5         8                   4.0    London      UK 2018-01-02  51.509865  -0.118092
+6        20                   NaN  Santiago      CL 2018-01-01        NaN        NaN
+7        22                   NaN  Santiago      CL 2018-01-02        NaN        NaN
+ 
+```
+
+## Remove and Delete Operations
+
+You can **Remove** entire partitions from the cube using the remove operation.
+
+```python
+>>> from kartothek.io.eager_cube import remove_partitions
+>>> datasets_after_removal = remove_partitions(
+...     cube=cube,
+...     store=store,
+...     ktk_cube_dataset_ids=["latlong"],
+...     conditions=(C("country") == "UK"),
+... )
+>>> query_cube(
+...     cube=cube,
+...     store=store,
+... )[0]
+   avg_temp  avg_temp_country_min      city country        day   latitude  longitude
+0         8                   6.0   Dresden      DE 2018-01-01  51.050407  13.737262
+1         4                   4.0   Dresden      DE 2018-01-02  51.050407  13.737262
+2         6                   6.0   Hamburg      DE 2018-01-01  53.551086   9.993682
+3         5                   4.0   Hamburg      DE 2018-01-02  53.551086   9.993682
+4         6                   6.0    London      UK 2018-01-01        NaN        NaN
+5         8                   4.0    London      UK 2018-01-02        NaN        NaN
+6        20                   NaN  Santiago      CL 2018-01-01        NaN        NaN
+7        22                   NaN  Santiago      CL 2018-01-02        NaN        NaN 
+```
+
+You can also **Delete** entire datasets (or the entire cube).
+
+```python
+>>> from kartothek.io.eager_cube import delete_cube
+>>> datasets_still_in_cube = delete_cube(
+...     cube=cube,
+...     store=store,
+...     datasets=["transformed"],
+... )
+>>> query_cube(
+...     cube=cube,
+...     store=store,
+... )[0]
+   avg_temp      city country        day   latitude  longitude
+0         8   Dresden      DE 2018-01-01  51.050407  13.737262
+1         4   Dresden      DE 2018-01-02  51.050407  13.737262
+2         6   Hamburg      DE 2018-01-01  53.551086   9.993682
+3         5   Hamburg      DE 2018-01-02  53.551086   9.993682
+4         6    London      UK 2018-01-01        NaN        NaN
+5         8    London      UK 2018-01-02        NaN        NaN
+6        20  Santiago      CL 2018-01-01        NaN        NaN
+7        22  Santiago      CL 2018-01-02        NaN        NaN
+```
+
+## Additional Features of Cube
+
+*	**Multiple-datasets**: This was not possible with the existing multi-table (within a single dataset) feature present in kartothek.
```

Comment:
Please swap the first and the second sentence, otherwise the reader might be confused about what exactly was not possible with the multi-table approach.

Reply:
Updated.